### PR TITLE
Add template for order food

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #1343
[Problem] No template
[Solution] Creating a template

Signed-off-by: Kornelia Kobiela <k.kobiela@samsung.com>

### CODE REVIEW

File: examples/mobile/HomeApp/breakfast-menu.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+				</li>
+				<li class="ui-li-divider ui-li-has-checkbox">
+					<input name="gluten" type="checkbox" />
+					<div class="ui-li-text">Gluten Free</div>
+				</li>
+			</ul>
+		</div>
+		<footer class="ui-bottom-button">
+			<a href="#" class="ui-btn ui-btn-flat">Submit your order</a>
+		</footer>
```
Comment (lslachciak) [inline examples/mobile/HomeApp/breakfast-menu.html:56]: Submit order could go back to home app

**Comment by lslachciak** [comment]
I next steps it would be good to 
- include images used in PR 
https://github.com/Samsung/TAU/pull/1345
- show more details about selected breakfast type
- if the given type of food is selected in home app, this screen should show this type of food selected by default
- how about using dropdown list to select breakfast type instead of radio buttons - screen will not be so filled in such case
- allow to define number of guest

File: examples/mobile/HomeApp/breakfast-menu.html (hunk 1 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/breakfast-menu.html:56]: to `#main` page

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

File: examples/mobile/HomeApp/breakfast-menu.html (hunk 1 above)
Comment (singa2000) [reply examples/mobile/HomeApp/breakfast-menu.html:56]: Now it's ok.
But if you "Submit your order", would you go  the payment page or payment site, please?
It's just enough by image.

**Review by singa2000 — APPROVED** [review]

**Pull request merged**